backend/src/graphql/resolver/ContributionLinkResolver.test.ts

```diff
@@ -246,6 +246,7 @@ describe('Contribution Links', () => {
         })
 
         it('returns an error if missing startDate', async () => {
+          jest.clearAllMocks()
```

Comment:
I'm not sure why you add this? 
If it is to be sure that only the right error is thrown and logged, why don't you add it on every tests that checks on Throws and Logs? For example line 504 and 584 would then also need it.